tests/test-cases/cli/inspect/uri_safe_identifier_override/code-list-with-uri-safe-identifier-override.csv

```diff
@@ -0,0 +1,6 @@
+Uri Identifier,Label,Notation,Parent Uri Identifier,Sort Priority,Description
```

Comment:
Are we using any of these files in the `uri_safe_identifier_override` folder anymore? If not, can you remove them please?